lambdas/ssm-parameters/tests/ssm-parameters-handler.test.ts

```diff
@@ -0,0 +1,104 @@
+import { SsmParametersHandler } from "../src/ssm-parameters-handler";
+import { Context } from "aws-lambda";
+import { jest } from "@jest/globals";
+import { SSMProvider } from "@aws-lambda-powertools/parameters/ssm";
+
+describe("ssm-parameters-handler", () => {
+  const ssmParametersHandler = new SsmParametersHandler();
+  const ssmProvider = jest.mocked(SSMProvider).prototype;
+  jest.spyOn(ssmProvider, "getParametersByName");
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.resetAllMocks();
```

Comment:
Do you really need this `jest.resetAllMocks();` you already have `clearAllMocks();`